File: chapter_one/CF_PRACTICE/791A.c
// # https://codeforces.com/problemset/problem/791/

#include <stdio.h>

int main()
{
  int a;
  int b;
  int x;

  scanf("%d %d", &a, &b);

  for(x = 0; a <= b; x++){
    a = a * 3;
    b = b * 2;
  }
  printf("%d", x);

  return 0;
}

File: ONE_arch/chapter_one/CODEFORCES/CF_PRACTICE/791A.py
# https://codeforces.com/problemset/problem/791/A

a,b = [int(f) for f in input().split()]

x = 0

while True:
    if a > b:
        break
    else:
        a = a * 3
        b = b * 2
        x += 1
print(x)
